scripts/docker/readme.md

```diff
@@ -1,104 +1,201 @@
+Or you can open the `$ASTROBEE_WS/src` folder through the VSCode graphical interface, and you should then see a popup dialog from the Dev Containers plugin. Click the "Reopen in Container" button.
 
-## Optional: Build the docker image
+Either way, the Dev Containers plugin will download a pre-built Astrobee Robot Software Docker image from our official repository, start it running, and provide you with a development environment running inside the container.
 
-The fastest way to start running the software is to fetch a remote
-docker image by using the `--remote` option with the `run.sh` script
-described below. Building the docker images is not required.
+You can manage your Dev Containers configuration using the files in the `.devcontainer` folder at the top level. For example, you can select a different Docker image to install from [the list on GitHub](https://github.com/nasa/astrobee/pkgs/container/astrobee) using the `FROM` command in the `Dockerfile`.
 
-However, if you need to build a local copy of the docker images, run:
+## Interactively develop inside the Docker container
 
-    ./scripts/docker/build.sh
+You can start by selecting `View->Terminal` in the VSCode graphical interface. This will display a terminal session inside the Docker container where you can run arbitrary commands. Your container will persist throughout your VSCode session, and changes you make using the VSCode editor will be reflected inside the container, making it easy to do quick interactive edit/build/test cycles.
 
-By default, the build script will automatically detect your host's
-Ubuntu OS version and configure the docker image to use the same
-version using the Dockerfile variables `UBUNTU_VERSION`,
-`ROS_VERSION`, and `PYTHON`.
+(Going forward, we could add a lot of tips here about how best to use VSCode inside the container.)
 
-However, there is no requirement for the host OS and the docker image
-OS to match.  You can override the default and select a specific
-docker image Ubuntu version by specifying `--xenial`, `--bionic`, or
-`--focal` for Ubuntu 16.04, 18.04, or 20.04 docker images,
-respectively.
+# Option 2: Using the Docker support scripts
 
-For more information about all the arguments:
+## Quick start run
 
-    ./scripts/docker/build.sh -h
+You can install Astrobee Robot Software inside a Docker container and run a software simulation with:
 
+```bash
+cd $ASTROBEE_WS/src
+./scripts/docker/run.sh --remote
+```
 
-## Run the Astrobee simulator in the container
+Invoking `run.sh --remote` calls Docker with the right arguments such that it will download the latest pre-built complete Astrobee Robot Software image from our official Docker image repository, start it as a container, and invoke a default command inside the container (which runs the software simulation).
+
+## Quick start build (if needed)
+
+The fastest way to recompile the ARS software to reflect your local changes is:
```

Comment:
Why use `ARS`? We don't use this anywhere else and no explanation for the acronym is given. I suggest replacing it with `Astrobee` software (I see 15 instances to this in the PR, this comment applies to all)

Reply:
I just saw that apparently ARS is a thing xD Feel free to ignore this even if it makes it less readable imho

Reply:
I appreciate this feedback. I'm torn because I don't like technical jargon, but "Astrobee Robot Software" is the actual name of our software package (title of the main README), and it also felt strange to keep writing it out all the time. Sometimes we informally write "FSW" which is a common aerospace acronym, but in my opinion too jargon-y if we fail to define it, and also not technically correct (lots of the stuff in the software package is not flight software at all).

So maybe it's a weird response, but my take on how to address your comment is to use proper technical writing style: introduce the acronym the first time we use it, and use it consistently (ironically, more often) after that.